lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced/search-date-range-advanced.component.spec.ts

```diff
@@ -0,0 +1,246 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { TranslateModule } from '@ngx-translate/core';
+import { ContentTestingModule } from '../../../../testing/content.testing.module';
+import { SearchDateRangeAdvancedComponent } from './search-date-range-advanced.component';
+import { endOfDay, endOfToday, formatISO, startOfDay, startOfYesterday } from 'date-fns';
+import { Validators } from '@angular/forms';
+
+describe('SearchDateRangeAdvancedComponent', () => {
+    let component: SearchDateRangeAdvancedComponent;
+    let fixture: ComponentFixture<SearchDateRangeAdvancedComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [SearchDateRangeAdvancedComponent],
+            imports: [
+                TranslateModule.forRoot(),
+                ContentTestingModule
+            ]
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(SearchDateRangeAdvancedComponent);
+        component = fixture.componentInstance;
+        component.field   = 'test-field';
+        component.dateFormat = 'dd-MMM-yy';
+        fixture.detectChanges();
+        component.form.setValue({
+            dateRangeType: component.DateRangeType.ANY,
+            inLastValueType: component.InLastDateType.DAYS,
+            inLastValue: null,
+            betweenStartDate: null,
+            betweenEndDate: null
+        });
+        spyOn(component.updatedQuery, 'emit');
+        spyOn(component.updatedDisplayValue, 'emit');
+    });
+
+    const getElementByDataAutomationId = (dataAutomationId) => {
+        return fixture.debugElement.query(By.css(`[data-automation-id="${dataAutomationId}"]`)).nativeElement;
+    };
+
+    const enterValueInInputField = async (inputElementId: string, value: string) => {
```

Comment:
You don't need async here, please remove also await before each usage of enterValueInInputField